sns_ik_lib/matlab/runTest_snsIk_vel_cs.m

```diff
@@ -0,0 +1,129 @@
+function result = runTest_snsIk_vel_cs(solver, nTest, optTol, cstTol, fid)
```

Comment:
Probably not worth doing at this point, but eventually it would be nice to combine all of the velocity tests into a single test function, reducing a lot of the code duplication below. For now, I think that it is not worth the added development time to implement.

Reply:
Good suggestion. I will keep that in mind for the next iteration.